src/content/fr/2019/accessibility.md

```diff
@@ -0,0 +1,309 @@
+
+### Zoom et mise à l’échelle des pages
+
+Utiliser une [taille de police lisible](https://accessibleweb.com/question-answer/minimum-font-size/) et [une taille de cible raisonnablement grande](https://www.w3.org/WAI/WCAG21/quickref/#target-size) aide les utilisateurs à lire et à interagir avec votre site Web. Mais même les sites Web qui suivent parfaitement toutes ces directives ne peuvent pas répondre aux besoins spécifiques de chaque visiteur. C’est pourquoi les fonctionnalités de l’appareil telles que le pincement au zoom et la mise à l’échelle sont si importantes&nbsp;: elles permettent aux utilisateurs de modifier vos pages pour répondre à leurs besoins. Ou dans le cas de sites particulièrement inaccessibles utilisant de minuscules polices et boutons, cela donne aux utilisateurs la possibilité même d’utiliser le site.
+
+Il existe de rares cas où la désactivation de la mise à l’échelle est acceptable, comme lorsque la page en question est un jeu web utilisant des commandes tactiles. Si laissé activé dans ce cas, les téléphones des joueurs feront un zoom avant et arrière à chaque fois que le joueur tapotera deux fois sur le jeu, ce qui — ironiquement — le rendra inaccessible.
+
+De fait, les développeurs ont la possibilité de désactiver cette fonctionnalité en définissant l’une des deux propriétés suivantes dans la [balise meta viewport](https://developer.mozilla.org/fr/docs/Mozilla/Mobile/Balise_meta_viewport)&nbsp;:
+
+1. `user-scalable` mis à `0` ou `no`
+
+2. `maximum-scale` mis à `1`, `1.0`, etc.
+
+Malheureusement, les développeurs ont tellement abusé de cela que près d’un site sur trois sur mobile (32,21%) désactive cette fonctionnalité, et Apple (à partir d’iOS 10) ne permet plus aux développeurs Web de désactiver le zoom. Safari mobile [ignore simplement la balise](https://archive.org/details/ios-10-beta-release-notes). Tous les sites, quels qu’ils soient, peuvent être zoomés et mis à l’échelle sur les nouveaux appareils iOS.
+
+<figure>
+  <a href="/static/images/2019/accessibility/fig2.png">
+    <img src="/static/images/2019/accessibility/fig2.png" alt="Figure 2. Pourcentage de sites qui désactivent le zoom et la mise à l’échelle par rapport au type d’appareil." aria-labelledby="fig2-caption" aria-describedby="fig2-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSG3DTnx7j-YT1hnQpQYjDRD-rCSF1dXbgva-iJQZKdCKIt34ojGMDRhx74fF93CpPg7oGW_C68fWGT/pubchart?oid=2053904956&amp;format=interactive">
+  </a>
+  <div id="fig2-description" class="visually-hidden">Graphes verticaux en pourcentages, allant de 0 à 80 de 20 en 20, par rapport au type d’appareil, regroupées en ordinateur de bureau et mobile. Bureau activé&nbsp;: 75,46%&nbsp;; Bureau désactivé&nbsp;: 24,54%&nbsp;; Mobile activé&nbsp;: 67,79%&nbsp;; Mobile désactivé&nbsp;: 32,21%.</div>
```

Comment:
Do we need these nbsp; in `class="visually-hidden"` pieces?

Guessing is shouldn't matter either way and maybe easier to be consistent so probably alright but thought I'd ask the question.

Reply:
Well, in fact, I was reading everything in markdown preview, just to be sure to read even `visually-hidden` ones, so I didn't make any difference ^^

Reply:
Well, I would say this can be kept, I'm guessing at any case images are not displayed or CSS not loaded, this would be a comfort sugar :)